### ISSUE TITLE
Decide what to do about " and ' 

### ISSUE THREAD

**Issue by erik-brangs** [tomboy-notes/tomboy-ng#68]
During the conversation for #66 it was mentioned that ``'`` and ``"`` are not treated specially by Tomboy. I've got a note with ``"`` in the title in the ``<note-deletions>`` in my ``manifest.xml``. This seems to throw off the tomboy-ng XML parser. It causes sync to fail.

**Comment by davidbannon** [comment]
Right !  Looks like Ben wins !
My own tests indicated it was not necessary and threatened compatibility with Tomboy, looks like I got that wrong. I'll go over that again and be a bit more careful.
I only tested on Linux, Erik, are you working on Windows ?
Its a trivial fix but one we need to know the implications of before getting too excited ....

**Comment by erik-brangs** [comment]
The problem occurred on Windows. However, I also reproduced it on Linux with another note with ``"`` in the title.

**Comment by davidbannon** [comment]
Thank Erik, I have tested, again, this time very carefully.
Both double and single inverted commas, in title, no problem on Linux, windows and Mac. Have tried with odd and even number of inverted commas.

Loading, Indexing all work fine, its only when I come to sync a deleted note its an issue.

Have gone back to Tomboy, it definitely leaves both double and single alone.
```
  <title>Note all "notes" 'are' "the 'same" you' know</title>
  <text xml:space="preserve"><note-content version="0.1">Note all "notes" 'are' "the 'same" you' know

I really don't know what to do here. Feel I should remain Tomboy complient but its clearly wrong to do so....

**Comment by davidbannon** [comment]
OK, I have pushed out changes that ensure we only write munged versions of ' and " to local manifest but made no change to what is written in an actual note. As Tomboy does not share local manifest file, that should not affect compatibility between tomboy-ng and Tomboy. I hope.
I'll push up some more fresh binaries later tonight.

Thanks heaps Erik for sticking with this !

**Comment by davidbannon** [comment]
Erik, 0.17 binaries now available from v0.17 release (as yet unannounced as its missing the 32bit RPM). https://github.com/tomboy-notes/tomboy-ng/releases/tag/v0.17
Should solve this problem.
Thanks for your help Erik.
